sources/v1.30.0/proposal_1_upgrade_framework/0-gas-schedule.move

```diff
@@ -0,0 +1,507 @@
+// Script hash: ed7acdb6 
+// Source commit hash: 46bf426662e4dc1b9e7317fc7fc66594bfaf6edb
+
+// Gas schedule upgrade proposal
+//
+// Old Gas Schedule Hash (Sha3-512): f187e875ea2d4386bd48d91fc9fd2680dce17f1c4756cc7aa27e8fee8c6fa981f3b00ec96849470f8859bb9d4dd0e2a800822eba6d0c3b37d9cf2392564d28f0
+// Changes
+//   Feature version: 33 -> 34
+//   Parameters
+//         txn.max_num_dependencies :  512 -> 768
```

Comment:
Is this intended? why do we want to increase dependency limit?

Reply:
@georgemitenkov incremented this parameter here: https://github.com/aptos-labs/aptos-core/commit/28b9b7ad6665027d5651fe07203d36bd9fc63d71#diff-c96720602dd9322e65f8ce3d37cd9138af7c3d2f5dccc2fd7030992239b7e96dR252

I'm not sure about the reasoning behind the change though.